Library/Homebrew/formula.rb

```diff
@@ -3374,9 +3374,11 @@ def freeze
     # desc "Example formula"
     # ```
     #
-    # @!attribute [w] desc
     # @api public
-    attr_rw :desc
+    sig { params(val: T.nilable(String)).returns(T.nilable(String)) }
+    def desc(val = nil)
```

Comment:
```suggestion
    sig { params(val: String).returns(T.nilable(String)) }
    def desc(val = T.unsafe(nil))
```

So we don't allow explicit `desc nil`.

Reply:
@dduugg, the same goes for all the other ones. Sorry if that was too implicit. 😉

Reply:
Ah, I'm dense like that. I was just maintaining the existing contracts built default. Will clean up others in a follow-up commit.